machine/alphanetworks/alphanetworks_snh60a0_640f/kernel/drivers-base-dd-alpha-snh60a0-640f.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/drivers/base/dd.c b/drivers/base/dd.c
```

Comment:
Please do not include the same patch, over and over for each machine.

You can share patches across machines by putting them in a top level `machine/<vendor>/kernel` directory.

For example see d43c0007c0ad5e901ae1f0d960b5f1fb0bacb923 which adds a common kernel patch used by multiple accton machine.